payloads/library/GitBunnyGit/payload.txt

```diff
@@ -18,19 +18,19 @@ LED R G B
 ATTACKMODE RNDIS_ETHERNET 
 #ATTACKMODE ECM_ETHERNET
 
-source bunny_helpers.sh
-
 # Set your desired repo url and branch if you're not looking to update from mainline
 git_repo="https://github.com/hak5/bashbunny-payloads.git"
 git_branch="master"
 payloads_dir="/root/udisk"
-log_file="/root/udisk/payloads/$SWITCH_POSITION/git.log"
+log_file="/root/udisk/git.log"
+
+echo " ---------  Git Bunny Git [started] -----------" >> $log_file
 
-echo "Git Bunny Git" > $log_file
+[[ ! -d "$payloads_dir/payloads" ]] && { LED R 200; echo "Could not mount filesystem" >> $log_file; exit 1; }
```

Comment:
mounting happens on boot, well before the OS starts receiving commands. 
if you _REALLY_ want to check this is mounted, then actually check by grepping the `mount` command or something. but it really isn't necessary